flare/bin/flare.py

```diff
@@ -11,27 +10,51 @@
 APP_NAME = "flare"
 
 
+def get_flare_api_client(
+    *,
+    api_key: str,
+    tenant_id: t.Union[int, None],
+) -> FlareApiClient:
+    api_client = FlareApiClient(
+        api_key=api_key,
+        tenant_id=tenant_id,
+    )
+    current_user_agent: t.Union[str, bytes] = (
+        api_client._session.headers.get(
+            "User-Agent",
+        )
+        or ""
+    )
+    if isinstance(current_user_agent, bytes):
+        current_user_agent = current_user_agent.decode()
+    api_client._session["User-Agent"] = f"{current_user_agent} flare-splunk"
+    return api_client
+
+
 class FlareAPI(AuthBase):
-    def __init__(self, *, app):
+    def __init__(self, *, app: t.Any) -> None:
         # Should be able to use app.service.storage_passwords.get(),
         # but I can't seem to get that to work. list() works.
-        server_key: Optional[str] = None
-        tenant_id: Optional[str] = None
+        api_key: t.Optional[str] = None
+        tenant_id: t.Optional[int] = None
```

Comment:
How did this even work? FlareApiClient requires an int for the tenant id, not a string.

Reply:
If we're changing the variable, we should also change the storage name: `if item.content.username == "serverkey":` < these references to serverkey.

Reply:
Yes we should. "serverkey" does not really make it clear this is an api key.